### PR TITLE
Create go present slide scaffold and continuous Appengine deployment

### PR DESCRIPTION
**Pull request by juliaogris**
Appengine deployable scaffold has been created according to instructions on
https://godoc.org/golang.org/x/tools/cmd/present
- copy of `app.yaml`
- copy of ${GOAPTH}/src/github.com/golang/tool/cmd/present/* in repo root

Continuous deployment has been set up in cloudbuild.yaml to automatically
update appengine deployment at https://gotraining.appspot.com/

(apologies if this is sloppy, gotta rush to school pickup now and got a hard deadline of flights next weekend).

### CODE REVIEW

File: .gcloudignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+# Test binary, build with `go test -c`
+*.test
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
```
Comment (camscale) [inline .gcloudignore]: nit: no eol at eof

File: app.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- url: /static
+  static_dir: static
+- url: /.*
+  script: auto
+
+# nobuild_files is a regexp that identifies which files to not build.  It
+# is useful for embedding static assets like code snippets and preventing
+# them from producing build errors for your project.
+
+nobuild_files: content/
```
Comment (camscale) [inline app.yaml]: nit: no eol at eof

File: cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Set gcloubuild trigger for master branch only
+steps:
+- id: appengine_deployment 
+  name: 'gcr.io/cloud-builders/gcloud' 
+  args: ['app', 'deploy'] 
+  timeout: '600s'
```
Comment (camscale) [inline cloudbuild.yaml]: nit: no eol at eof

File: content/gotraining.slide [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+* Introduction
+
+* Agenda
+
+# * Go history
+
+# * Why Go at ANZ
+
+# * Ping server Java Spring Boot vs Go
```
Comment (camscale) [inline content/gotraining.slide]: nit: no eol at eof

**Review by camscale — DISMISSED** [review]
LGTM. you can fix the nits later if you want.

**Comment by juliaogris** [comment]
So sloppy, sorry... thought my editor was set up to always put EOL at EOF of open files.

**Comment by juliaogris** [comment]
PTAL @camh-anz

File: cloudbuild.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (camscale) [inline cloudbuild.yaml:7]: nit: editor still not right. you can be a perfectionist or you can just merge now.

**Review by camscale — APPROVED** [review]

File: cloudbuild.yaml (hunk 5 above)
Comment (juliaogris) [reply cloudbuild.yaml:7]: I can't believe that, apologies. Perfectionism limit reached, will fix up next time sorry

**Pull request merged**